frontend/src/metabase/core/components/EditableText/EditableText.styled.tsx

```diff
@@ -19,14 +19,15 @@ export const EditableTextRoot = styled.div<EditableTextRootProps>`
 
   &:hover,
   &:focus-within {
-    border-color: ${props => (props.isDisabled ? "" : color("border"))};
+    border-color: ${props =>
+      props.isDisabled ? "" : "var(--mb-color-border)"};
```

Comment:
Do you think this reads a bit better once we swap the condition a bit?
```suggestion
      !props.isDisabled ?? "var(--mb-color-border)"};
```